yarn.lock

```diff
@@ -8043,7 +7874,12 @@ lodash.words@^3.0.0:
   dependencies:
     lodash._root "^3.0.0"
 
-lodash@4.17.21, lodash@^4.13.1, lodash@^4.14.2, lodash@^4.17.10, lodash@^4.17.11, lodash@^4.17.12, lodash@^4.17.13, lodash@^4.17.14, lodash@^4.17.15, lodash@^4.17.2, lodash@^4.17.4, lodash@^4.17.5, lodash@^4.2.0, lodash@^4.3.0:
+lodash@4.17.15:
+  version "4.17.15"
```

Comment:
This is not good due to a CVE required us to update lodash to >= 4.17.21.
Check https://github.com/oVirt/ovirt-web-ui/pull/1413
Please revert back this section if possible for using 4.17.21 instead.

Reply:
Fixed in #1437